src/main/java/library/app/spring/controller/IndexController.java

```diff
@@ -0,0 +1,14 @@
+package library.app.spring.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+@RequestMapping("/index")
+public class IndexController {
+    @GetMapping("/")
```

Comment:
Doesn't it redundant? I mean "/"

Reply:
I tested it. in fact, it works on both ways